packages/client/tests/functional/_example/tests.ts

```diff
@@ -47,6 +47,7 @@ testMatrix.setupTestSuite(
     })
 
     test('getTestSuiteSchema', () => {
+      // @ts-expect-error
```

Comment:
Is this due to that `Object.keys(...)[0]` referenced elsewhere? What's the actual type error?

Reply:
This is a small thing actually, don't worry about it.